typescript/web/src/pages/[workspaceSlug]/datasets/[datasetSlug]/images/index.tsx

```diff
@@ -29,12 +29,6 @@ export const datasetDataQuery = gql`
       imagesAggregates {
         totalCount
       }
-      labelsAggregates {
```

Comment:
Why are you removing this part ? 🤔

Reply:
because it's never ever used, and makes extra SQL queries